content/pages/privacy-policy.md

```diff
@@ -0,0 +1,105 @@
+title: Mixxx Privacy Policy
+slug: privacy-policy
+
+## Introduction
+
+Mixxx is a non-profit project, managed by volunteers.
+
+There is no business-model around Mixxx, and the only revenue we receive is through donations.
+
+For privacy policies and conformance to GDPR, a few distinctions will be done between:
+
+- Mixxx application
+- Main Mixxx website
+- Community Mixxx websites
+
+For each of the previous software/websites, we'll detail the different privacy policies.
+
+## Mixxx application
+
+The Mixxx application doesn't use any user account, and does not collect any user data, when running.

```

Comment:
We have user accounts for broadcasting and we may have in future accounts for Last.fm or others.

I think we want to say something lightly diffrent: 

```
The Mixxx application does not utilize user accounts to collect any user data while running. However, in certain specific functions, collecting and sharing limited personal information is required. This applies to the following three precise cases:
```

Than I would remove the bullet point list, but number the three cases to avoid duplication.

Reply:
Added broadcasting and removed Musicbrainz/Metadata dupliation